airflow/api_fastapi/core_api/routes/public/dags.py

```diff
@@ -69,6 +75,25 @@ def get_dags(
     only_active: QueryOnlyActiveFilter,
     paused: QueryPausedFilter,
     last_dag_run_state: QueryLastDagRunStateFilter,
+    dag_run_start_date_range: Annotated[
+        RangeFilter, Depends(datetime_range_filter_factory("start_date", DagRun))
+    ],
+    dag_run_end_date_range: Annotated[
+        RangeFilter, Depends(datetime_range_filter_factory("end_date", DagRun))
+    ],
```

Comment:
```suggestion
        RangeFilter, Depends(datetime_range_filter_factory("start_date", DagRun))
    ],
    dag_run_end_date_range: Annotated[
        RangeFilter, Depends(datetime_range_filter_factory("dag_run_end_date", DagRun))
    ],
```
```suggestion
        RangeFilter, Depends(datetime_range_filter_factory("start_date", DagRun))
    ],
    dag_run_end_date_range: Annotated[
        RangeFilter, Depends(datetime_range_filter_factory("end_date", DagRun))
    ],
```
Same for start date.

Reply:
Done.